ckanext/datajson/harvester_datajson.py

```diff
@@ -50,6 +50,16 @@ def load_remote_catalog(self, harvest_job):
             catalog_values = datasets.copy()
             datasets = catalog_values.pop("dataset", [])
 
+            # hotfix to deal with an unexpected domain redirection
+            for k, v in catalog_values.items():
+                catalog_values[k] = v.replace("https://project-open-data.cio.gov/v1.1/schema", "https://resources.data.gov/schemas/dcat-us/v1.1/schema")
+    
+        for dataset in datasets:
+            if dataset.get("license") == "https://project-open-data.cio.gov/unknown-license":
+                dataset["license"] = "https://github.com/project-open-data/project-open-data.github.io/blob/master/unknown-license.md"
+            elif dataset.get("license") == "https://project-open-data.cio.gov/open-licenses":
+                dataset["license"] = "https://resources.data.gov/open-licenses/"
```

Comment:
I know this is meant as a temporary fix but I really want to avoid this. I'm afraid if we modify the third-party metadata like this, we find ourselves on a slippery slope. By modifying the agency's metadata, we're sort of tampering with it, diminishing it's authenticity.

As open data stewards, we want to maintain the authenticity and integrity in our chain of custody of data as much as possible using reproducible and well-known methods.